lib/modules/manager/gomod/artifacts.ts

```diff
@@ -59,6 +59,7 @@ function getGitEnvironmentVariables(): NodeJS.ProcessEnv {
     PlatformId.Azure,
     PlatformId.Bitbucket,
     PlatformId.BitbucketServer,
+    PlatformId.CodeCommit,
```

Comment:
Are you shure go works with codecommit?

Reply:
dont see a reason not to
codecommit is a repo holder why wouldnt it?

Reply:
Let's leave it like the others, it hopefully should behave the same